app/models/experimentation/experiment.rb

```diff
@@ -29,9 +29,8 @@ def self.candidate_patients
         .where(["experiments.end_date < ? OR experiments.id IS NULL", ExperimentControlService::LAST_EXPERIMENT_BUFFER.ago]).references(:experiment)
     end
 
-    def group_for(uuid)
-      hash = Zlib.crc32(uuid) % treatment_groups.length
-      treatment_groups.find_by(index: hash)
+    def random_treatment_group
+      treatment_groups[rand(0..treatment_groups.count - 1)]
```

Comment:
`treatment_groups.sample` would be simpler here I think

Reply:
Didn't think of that. Good idea. I'll do that.